lib/pbench/test/functional/server/conftest.py

```diff
@@ -47,7 +47,7 @@ def register_test_user(server_client: PbenchServerClient):
     ), f"Register failed with {response.json()}"
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def login_user(server_client, register_test_user):
```

Comment:
Nobody would log this user out during a test...would they?

Reply:
Well, they could. I promoted this to "module" because the default level was logging in repeatedly and sometimes hitting within the "window of uniqueness" and prompting login's annoying `CONFLICT` message. I wanted to promote it to "session" ... and tried ... but I'd have to promote `server_client` as well and decided not to get into that.

Reply:
If it doesn't bother you, I'll try not to let it bother me.  😉